pkg/maintainer/spv/spv.go

```diff
@@ -165,17 +166,10 @@ func (sm *spvMaintainer) proveTransactions(
 			transactionHashStr,
 		)
 
-		accumulatedConfirmations, err := sm.btcChain.GetTransactionConfirmations(
-			transaction.Hash(),
-		)
-		if err != nil {
-			return fmt.Errorf(
-				"failed to get transaction confirmations: [%v]",
-				err,
-			)
-		}
-
-		isProofWithinRelayRange, requiredConfirmations, err := getProofInfo(
+		isProofWithinRelayRange,
```

Comment:
Can we format this call like this:
```
isProofWithinRelayRange, accumulatedConfirmations, requiredConfirmations, err := getProofInfo(
    transaction.Hash(),
    sm.btcChain,
    sm.spvChain,
    sm.btcDiffChain,
)
```
Even if it exceeds the maximum line length it should be more readable than the current form. WDYT?

Reply:
Done in e982bd06521a6293bde6ae37ddf100836abd7b00.